config/zones.json

```diff
@@ -95,15 +95,18 @@
       ]
     ],
     "capacity": {
-      "biomass": 474,
+      "biomass": 500,
       "coal": 598,
-      "gas": 4466,
-      "hydro": 8590,
-      "hydro storage": 3401,
+      "geothermal": 1,
+      "storage": 2439,
```

Comment:
do you mean battery storage here?

Reply:
it's not completely clear if it's battery storage. I think it's all types of storage excluded hydro storage.

I don't know how to handle this?

Reply:
Hmm that's tricky. The thing is that in order for the UI to be consistent, capacities should be tied to the data we're receiving from ENTSO-E.
Currently, ENTSO-E just has hydro storage. It would be interesting to understand what other type of storage is available. Because the storage you mention is non negligible, we need to know what it represents.

Reply:
@Bohne13 "Speicher", in this case, refers to hydro storage, without the capability to pump back water to the storage (so non-pumped storage).
These hydro plants have natural sources of water (rivers, glaciers, ...). They are also referred to as "Saisonspeicher", because usually they fill during springtime and summer. 
In the past, we always combined this category with run-of-river/usual hydro. 
So you must add the 2439 MW to the hydro category, then it's fine.